integration_tests/fixtures/fileUpload/larger-than-100kb.csv

```diff
@@ -0,0 +1,10000 @@
+﻿Prison number
+TESTDATA

```

Comment:
Could this file be generated by the test and discarded afterwards?

Reply:
Could do sure. Would you like me to do that?

Reply:
Maybe as another step? Saves a 100kb file in source code. Fine for now.